atlasdb-workload-server-antithesis/build.gradle

```diff
@@ -8,4 +8,36 @@ docker {
     name "palantirtechnologies/${project.name}:${imageVersion}"
     tags 'latest', 'snapshot'
     files 'var/docker-compose.yml', 'var/compact.sh', 'var/flush.sh'
+}
+
+dependencies {
+    testImplementation project(":atlasdb-workload-server-distribution")
+    testImplementation project(":timelock-server-distribution")
+
+    testImplementation 'com.palantir.docker.compose:docker-compose-rule-core'
+    testImplementation 'com.palantir.docker.compose:docker-compose-junit-jupiter'
+    testImplementation 'com.palantir.safe-logging:safe-logging'
+    testImplementation 'io.dropwizard:dropwizard-testing'
+    testImplementation 'org.assertj:assertj-core'
+    testImplementation 'org.awaitility:awaitility'
+    testImplementation 'org.junit.platform:junit-platform-suite'
+}
+
+task prepareDockerImages (type: Exec) {
+    dependsOn ':atlasdb-workload-server-distribution:dockerTag'
+    dependsOn ':timelock-server-distribution:dockerTag'
+    dependsOn dockerTag
+    commandLine 'sh', './tag-test-images.sh', project.version.toString()
+}
+
+tasks.named("test") {
+    dependsOn prepareDockerImages
+}
+
+test {
+    include '**/AntithesisDockerTest.class'
```

Comment:
might be missing something: Do we need this? I _thought_ we'd run all tests by default

Reply:
Ah, that's true! Reminiscent from when I was trying to have a different task name